src/ucp/tag/tag_match.h

```diff
@@ -41,8 +41,12 @@ typedef struct ucp_tag_match {
     /* Tag offload fields */
     ucs_queue_head_t          offload_ifaces; /* Interfaces which support tag offload */
     ucs_queue_head_t          sync_reqs;      /* Outgoing sync send requests */
-    size_t                    post_thresh;    /* Messages are posted to the transport
-                                                 starting from this threshold. */
+    size_t                    thresh;         /* All messages above the threshold are posted
```

Comment:
IMHO it is not very clear what difference between below vs above the threshold. Please be verbose .